### PR TITLE
CircleCI runs ethereum integration test also against parity

### PR DESCRIPTION
**Pull request by se3000**

### CODE REVIEW

File: integration/common [hunk 1]
```diff
@@ -80,6 +80,20 @@ launch_gethnet() {
   title "Geth is running."
 }
 
+launch_parity() {
+  printf -- "\033[34m%-50s\033[0m" "Checking for running ethereum node"
```
Comment (j16r) [inline integration/common]: Copy and paste error

File: integration/common [hunk 2]
```diff
@@ -80,6 +80,20 @@ launch_gethnet() {
...
+  if curl -s $ETH_HTTP_URL >/dev/null; then
```
Comment (j16r) [inline integration/common]: Wrong URL

File: integration/common [hunk 3]
```diff
@@ -80,6 +80,20 @@ launch_gethnet() {
...
+    printf -- "[\033[32mrunning\033[0m]\n"
+    return
+  fi
+  printf -- "[\033[31mstopped\033[0m]\n"
+
+  title "Starting parity..."
+  $SRCROOT/internal/bin/devnet &>$SRCROOT/integration/devnet.log &
+  waitForResponse $ETH_HTTP_URL
```
Comment (j16r) [inline integration/common]: Wrong URL

Comment (dimroc) [reply integration/common]: Can you suggest what the correct URL should be?

File: integration/common (hunk 1 above)
Comment (dimroc) [reply integration/common]: What should be the correct copy? Seems almost identical to line 68 which was the intention.

**Pull request merged**